crates/sui-core/src/checkpoints/checkpoint_executor/mod.rs

```diff
@@ -516,7 +516,7 @@ impl CheckpointExecutor {
 
 // Logs within the function are annotated with the checkpoint sequence number and epoch,
 // from schedule_checkpoint().
-#[instrument(level = "debug", skip_all)]
+#[instrument(level = "debug", skip_all, fields(seq = ?checkpoint.sequence_number(), epoch = ?epoch_store.epoch()))]
```

Comment:
Does level need to be error?

Reply:
Hmm, good point. This replaces all the `debug!()` below, so I guess it's a no-op currently. But I'm not sure why schedule_checkpoint uses `error`. Do you have any preference?

Reply:
I'm unfamiliar with setting instrument level. My understanding is that we should use `error` to avoid the annotations and span being filtered out at default tracing level. If this annotation is a no-op right now, we should change the instrument level to error.

Reply:
Resolved offline. Keep `DEBUG` as convention.